app/views/user_email_mailer/results_email.html.erb

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Email letter</title>
+  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+  <style>
```

Comment:
what is the reason to have this style tag implemented separately from others in email_page.css??

Reply:
Hi Vitalii. The reason why need and why this code was left in the results_email.html.erb and response_invite.html.erb concluded in that cause email post servers Google and OUTLOOK  without this code make display e-mail letters differently btw them and with design mistakes (such as no buttons, icon etc). I waste a lot of time when I understood this.  Thanks to this decision all email display simallar